learn/advanced/indexation.md

```diff
@@ -42,13 +42,13 @@ If you encounter performance issues during the indexation we recommend trying th
     "id": 1,
     "title": "Pride and Prejudice",
     "author": "Jane Austin",
-    "genre": "romance"
+    "genres": "romance"
   },
   {
     "id": 2,
     "title": "Le Petit Prince",
     "author": "Antoine de Saint-Exupéry",
-    "genre": "adventure"
+    "genres": "adventure"
```

Comment:
FWIW, these examples are not linked to any real dataset (they appear to describe books, not movies). So we could probably revert these changes, not that it matters much 🤷🏻

Reply:
You’re right, thanks!